src/js/components/covid19/PublicLawPicker.jsx

```diff
@@ -5,21 +5,36 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import GlossaryLink from '../sharedComponents/GlossaryLink';
 
 const options = [
-    { title: 'All COVID-19 Funding Laws', value: 'all', term: 'disaster-emergency-fund-code-defc' },
-    { title: 'American Rescue Plan', description: 'Non-emergency Public Law 117-2', value: 'american-rescue-plan' },
-    { title: 'Learn more about filtering USAspending data by Public Law in our Data Sources & Methodology page.', value: 'dsm', className: 'dsm' }
+    {
+        title: 'All COVID-19 Funding Laws',
+        value: 'all',
+        term: 'disaster-emergency-fund-code-defc'
+    },
+    {
+        title: 'American Rescue Plan',
+        description: 'Non-emergency Public Law 117-2',
+        value: 'american-rescue-plan'
+    },
+    {
+        title: 'Learn more about filtering USAspending data by Public Law in our Data Sources & Methodology page.',
+        value: 'dsm',
+        className: 'dsm'
+    }
 ];
 
 const PublicLawPickerOption = ({
-    title, description, term, className
+    title,
+    description,
+    term,
+    className
 }) => (
     <li className={className}>
         <p>{title}</p>
         {description && (
             <p>
                 {description}
                 {term && (
-                    <GlossaryLink term={term} />
+                    <GlossaryLink term={term}/>
```

Comment:
This change is throwing a lint error: `error  A space is required before closing bracket  react/jsx-tag-spacing`
```suggestion
                    <GlossaryLink term={term} />
```